radicle-httpd/src/api/v1/projects.rs

```diff
@@ -202,7 +202,7 @@ async fn history_handler(
     });
 
     if fallback_to_head {
-        return Ok::<_, Error>((StatusCode::FOUND, Json(response)));
+        return Ok::<_, Error>((StatusCode::OK, Json(response)));
     }
 
     Ok::<_, Error>((StatusCode::OK, Json(response)))
```

Comment:
If we change the status codes to OK 200 I suggest to remove the entire `fallback_to_head` if branch

```diff
diff --git a/radicle-httpd/src/api/v1/projects.rs b/radicle-httpd/src/api/v1/projects.rs
index d81b756..852ca48 100644
--- a/radicle-httpd/src/api/v1/projects.rs
+++ b/radicle-httpd/src/api/v1/projects.rs
@@ -201,10 +201,6 @@ async fn history_handler(
         "stats":  repo.stats()?,
     });
 
-    if fallback_to_head {
-        return Ok::<_, Error>((StatusCode::FOUND, Json(response)));
-    }
-
     Ok::<_, Error>((StatusCode::OK, Json(response)))
 }
```

Reply:
fba250fa40a9764b67e9691e8bc4e5ff24c0d691